File: JavaScript/BinarySearch.js
let list=[];
for (let i=0; i < 100; i++) {
    list.push(i)
}

let l = 0;
let h = list.length - 1;
let m = l + Math.floor((h - l) / 2);
let target = Math.floor(Math.random() * (99 - 0 + 1) + 0);

while (true) {
    m = l + Math.floor((h - l) / 2);
    if (m > target) {
        h = m;
    } else if (m < target) {
        l = m;
    } else {
        break;
    }
}

File: Python/Binary_search.py
list = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

target = 8
h = list[len(list) - 1]
l = list[0]

while True:
    m = l + (h - l)//2
    if list[m] > target:
        h = m
    elif list[m] < target:
        l = m
    else:
        print(list[m])
        break
